src/commons/mail-message-renderer.jsx

```diff
@@ -79,11 +117,17 @@ const _TextMessageRenderer = ({ body, t }) => {
 	);
 };
 
-const _HtmlMessageRenderer = ({ msgId, body, parts, t }) => {
+const _HtmlMessageRenderer = ({ msgId, body, parts, t, participants }) => {
 	const divRef = useRef();
 	const iframeRef = useRef();
 	const [showQuotedText, setShowQuotedText] = useState(false);
-	// const settings = useUserSettings();
+
+	const settingsPref = useUserSettings()?.prefs;
+	const from = filter(participants, { type: 'f' })[0].address;
+	const domain = from.split('@')[1];
```

Comment:
I have doubt in finding a domain name like this, the better way is to split from the last index of '@' like
var email = "john.doe@email.com";
var domain = email.substring(email.lastIndexOf("@") +1);